ci/test-checks.sh

```diff
@@ -15,7 +15,7 @@ cargoNightly="$(readlink -f "./cargo") nightly"
 
 scripts/increment-cargo-version.sh check
 
-if [ -n "$CI" ] ; then
+if [ -z "${SOLANA_CI_ALLOW_STALE_CARGO_LOCK:+x}" ] ; then
```

Comment:
btw, I guess `if [[ ! -v SOLANA_CI_ALLOW_STALE_CARGO_LOCK ]]; then` do the trick as well

Reply:
oh yeah... missed `-v` in `help test`. i thought there was a more appropriate way to do this check.